src/types/Messages.ts

```diff
@@ -1,3 +1,3 @@
 export enum Message {
-  updatePlayer,
+  UPDATEPLAYER,
```

Comment:
GG sorry I was not clear, for these constants/enums, it should look like: `UPDATE_PLAYER`

Reply:
Done.